backend/api.test/Client/MissionTests.cs

```diff
@@ -657,28 +444,24 @@ public async Task GetNextRun()
         [Fact]
         public async Task ScheduleDuplicatMissionDefinitions()
         {
-            // Arrange - Initialise areas
-            string installationCode = "installationScheduleDuplicatMissionDefinitions";
-            string plantCode = "plantScheduleDuplicatMissionDefinitions";
-            string deckName = "deckScheduleDuplicatMissionDefinitions";
-            string areaName = "areaScheduleDuplicatMissionDefinitions";
-            (_, _, _, _) = await PostAssetInformationToDb(installationCode, plantCode, deckName, areaName);
-
-            // Arrange - Create mission definition
-            string robotUrl = "/robots";
-            var response = await _client.GetAsync(robotUrl);
-            Assert.True(response.IsSuccessStatusCode);
-            var robots = await response.Content.ReadFromJsonAsync<List<Robot>>(_serializerOptions);
-            Assert.NotNull(robots);
-            var robot = robots.Where(robot => robot.Name == "Shockwave").First();
+            // Arrange - Initialise area
+            var installation = await _databaseUtilities.ReadOrNewInstallation();
+            var plant = await _databaseUtilities.ReadOrNewPlant(installation.InstallationCode);
+            var deck = await _databaseUtilities.ReadOrNewDeck(installation.InstallationCode, plant.PlantCode);
+            var area = await _databaseUtilities.ReadOrNewArea(installation.InstallationCode, plant.PlantCode, deck.Name);
+
+            // Arrange - Robot
+            var robot = await _databaseUtilities.NewRobot(RobotStatus.Available, installation);
             string robotId = robot.Id;
-            string missionSourceId = "986"; // Corresponds to mock in ServiceMock.cs
+
+            string missionSourceId = "986";
```

Comment:
Could this be fetched from the mock itself somehow? Not a priority in this PR though

Reply:
Probably could yeah, will not fix in this PR.